test/unit/gateways/pay_junction_v2_test.rb

```diff
@@ -205,6 +206,21 @@ def test_failed_store
     assert_match %r{Card Number is not a valid card number}, response.message
   end
 
+  def test_add_address
+    post = {:card => {:billingAddress => {}}}
```

Comment:
Just a style preference, but since these keys are symbols, this line could be a bit shorter if you wanted:
```
post = {card: {billingAddress: {}}}
```

Reply:
This is another thing I copied from the Adyen gateway implementation. I'm not sure what is the convention across AM, but I'd prefer to do it however Spreedling's are doing it.

Reply:
This is most likely a case where older implementation are using older `=>` syntax and RuboCop simply hasn't caught up yet. I would support switching to the new `card:` hash syntax here.